src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+import java.util.Random;
+public class Ball {
+    private Color color;
+    private int number;
+
+    public Ball() {
+        this.color = new ColorSupplier().getRandomColor();
+        this.number = new Random().nextInt(100);
+    }
```

Comment:
Having one constructor with predefined values you will struggle expanding your application when you will need to create ball with custom values
```suggestion
    public Ball(Color color, int number) {
        this.color = color;
        this.number = number;
    }
```